frontend/src/scenes/insights/InsightTabs/TrendTab/TrendTab.scss

```diff
@@ -5,5 +5,22 @@
         display: flex;
         justify-content: space-between;
         padding-bottom: 4px;
+        align-items: center;
+    }
+
+    .new {
+        background-color: #66c2a5;
```

Comment:
Nit: this is a duplicate declaration of `return cssHSL(161, 43, 58)` in `lib/colors.ts`... these files could get out of sync if we add new states or change colors again. Would be nice to see CSS vars in both places.

Reply:
Yeah I agree they should both at least be in hex but I'm not sure about turning them into CSS vars in both places, how do I do that without having to declare CSS variables twice in both `global.scss` and `colors.ts`?